hyperspy/_components/gaussian2d.py

```diff
@@ -48,18 +50,21 @@ def __init__(self,
                  sigma_y=1.,
                  centre_x=0.,
                  centre_y=0.,
+                 rotation=0.,
```

Comment:
Will do, but I think math.sin and math.cos takes radians as input, ergo rotation will be in radians as well (unless I'm misunderstanding something).